helm-charts/templates/data-plane/gateway-components/gateway-runtime/jwks-domain-api-httproute.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
     api-version: "1.0.0"
 spec:
   hostnames:
-  - "api.am.wso2.com"
+  - "{{ .Values.wso2.apk.listener.hostname | default "api.am.wso2.com"}}"
```

Comment:
let's add all new helm variables to values.template file

Reply:
It's already added template file and readme